### PR TITLE
Fixes remote upload from app to docker

### LINKED ISSUE #3667: Error when uploading Asset Icon from Rotki Windows App into Docker
## Problem Definition
I use rotki with docker on latest version and i connect to this instance with my windows installation.

When i try to upload an icon to a self created asset i get an error on my windows maschine.
When i upload it directly in the docker instance (over web browser) it works fine.

Getting error status code 400

### System Description

Operating system: Windows 10
Rotki version: 1.21.3
Docker: 1.21.3

image

### PR DESCRIPTION
**Pull request by kelsos**
Closes #3667

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/services/rotkehlchen-api.ts [hunk 1]
```diff
@@ -170,7 +170,12 @@ export class RotkehlchenApi {
         },
         { validateStatus: validAccountOperationStatus }
       )
-      .then(handleResponse);
+      .then(value => {
+        if (value.status === 409) {
```
Comment (kelsos) [inline frontend/app/src/services/rotkehlchen-api.ts:174]: I am sneaking this in, this is for the frontend to treat a 409 (no user logged in) as a successful logout for the frontend and get you to the login screen.

Comment (yabirgb) [reply frontend/app/src/services/rotkehlchen-api.ts:174]: I was going to ask about it :+1:

Comment (kelsos) [reply frontend/app/src/services/rotkehlchen-api.ts:174]: I did some weird forced reloadings and I couldn't logout due to 409 :P

**Review by yabirgb — APPROVED** [review]
Seems to work now. LGTM

**Pull request merged**